### PR TITLE
chore: add toaster story

### PR DESCRIPTION
**Pull request by wmhilton**
I needed to mock up some screenshots of the toast mechanism.

### CODE REVIEW

File: src/__stories__/Toaster/index.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+      action = {
+        icon: text('action.icon', 'airplane') as IconName,
+        text: text('action.text', 'Action Text'),
+        onClick: StorybookAction('onClick action'),
+      };
+      props = {
+        ...(boolean('show action', false) && { action }),
+        intent: select('intent', Intent, 'primary'),
+        message: text('message', 'Hello world!'),
+      };
```
Comment (wmhilton) [inline src/__stories__/Toaster/index.tsx]: I found I had to re-run these or else they wouldn't be up-to-date. Also, Storybook doesn't let you use hooks, and frankly... I don't really know how knobs work. There's probably a better way.

Comment (P0lip) [reply src/__stories__/Toaster/index.tsx]: > I found I had to re-run these or else they wouldn't be up-to-date. 

Whatcha mean by knobs being not up to date?

Comment (wmhilton) [reply src/__stories__/Toaster/index.tsx]: Try commenting it out and see.

Comment (P0lip) [reply src/__stories__/Toaster/index.tsx]: Ah yeah, I see. Hmmm, in any case, it's not a big deal. Suggest leaving a comment on that in the code though.

**Pull request merged**